hack/boilerplate/boilerplate.py

```diff
@@ -207,7 +218,8 @@ def get_regexs():
     # company holder names can be anything
     regexs["date"] = re.compile(get_dates())
     # strip // +build \n\n build constraints
-    regexs["go_build_constraints"] = re.compile(r"^(// \+build.*\n)+\n", re.MULTILINE)
+    regexs["go_build_constraints"] = re.compile(
+        r"^(//(go:build| \+build).*\n)+\n", re.MULTILINE)
```

Comment:
Since we replaced all `+build` with `go:build` anyway, maybe we can shorten the regex and get rid of `+build` from the pattern?

Reply:
All this code I have taken from upstream autoscaler. I don't want to change the upstream code where we can avoid it.

Reply:
Makes sense.